data/transactions/logic/fields.go

```diff
@@ -149,6 +149,18 @@ const (
 	// Nonparticipation Transaction.Nonparticipation
 	Nonparticipation
 
+	// Logs Transaction.ApplyData.EvalDelta.Logs
+	Logs
+
+	// NumLogs len(Logs)
+	NumLogs
+
+	// EvalConfigAsset Transaction.ApplyData.EvalDelta.ConfigAsset
+	EvalConfigAsset
```

Comment:
I wanted to retain the exact name of the txn field "ConfigAsset" and "ApplicationID", but show that it is in EvalDelta (or an "Evaluation" effect).  It sounds like you both like the change though.  Might have to be "Application" for consistency?

Reply:
`CreatedApplicationID` sounds good to me.

I recognize the effort to make it clear this is an evaluation field, but since `Logs` don't have an "Eval" prefix either, I think it's better to have this simpler name.

Reply:
Ok, by me.